### PR TITLE
feat(dedicated.ip): add filter option based on ip version

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-9677
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ [n/a]

## Description

Add filter option based on ip version IPV4 and IPV6

## Related

- [x] TRANS-50977

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.html [hunk 1]
```diff
@@ -81,6 +81,23 @@
                     class="d-flex align-items-center"
                     data-ng-transclude="filters"
                 ></div>
+                <oui-dropdown data-placement="end">
+                    <oui-dropdown-trigger
+                        text="{{ FILTER_OPTIONS.FILTER }} "
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/list.html:86]:
```suggestion
                        data-text="{{ FILTER_OPTIONS.FILTER }} "
```

**Review by anooparveti — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.html (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/ip/components/list/list.html]: Done Anoop

File: packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -12,6 +12,8 @@
   "ip_ipv4": "IPv4",
   "ip_ipv6": "IPv6",
   "ip_ipblock": "IP Block",
+  "ipv4_filter": "Afficher Ies IPV4",
+  "ipv6_filter": "Afficher Ies IPV6",
```
Comment (ghyenne) [inline packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json:16]:
```suggestion
  "ipv4_filter": "Afficher les IPV4",
  "ipv6_filter": "Afficher les IPV6",
```
it isn't upper case I (i) but lowercase l (L)

**Review by ghyenne — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json (hunk 2 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json]: Done

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by mohammed-zahaf — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**